docs/features.md

```diff
@@ -0,0 +1,47 @@
+## Shop-easy platform features
+
+There are two user roles in our platform. They are
+    - Admin User
+    - Student User
+
+### Admin User:
+
+Admin has administrative previliges and responsible for pantry management such as Inventory management and order tracking etc.,
+
+### Student User:
+
+Student can access the pantry platform for convinent and user friendly purchases of the listed items.
+
+### Admin features:
+
+- **Admin authentication**: Admin login worflow with username and password.
+- **Inventory Management**: Manage inventory to add, update and delete items based on the category.
+- **Order Tracking**: Tracking student orders, their status and history.
+- **User Management**: Administrative previliges to add, update and delete students.
```

Comment:
In user management, we should be able to assign multiple levels of access to users and also track their activity.

Reply:
addressed